src/Steps/Release/PublishRelease.php

```diff
@@ -288,6 +290,132 @@ protected function updateGithubChangelog(OutputInterface $output, LibraryRelease
         $this->log($output, "Github API update failed for <info>" . $library->getName() . "</info>", "error");
     }
 
+    /**
+     * Use the GitHub API to get the release notes for this release
+     */
+    private function getReleaseNotes(OutputInterface $output, GithubClient $client, LibraryRelease $release): string
+    {
+        $this->log($output, 'Getting release notes from GitHub API');
+
+        // Get arguments to send to GitHub API
+        $version = $release->getVersion();
+        $args = [
+            'tag_name' => $version->getValue(),
+            'target_commitish' => "{$version->getMajor()}.{$version->getMinor()}",
+            'previous_tag_name' => $this->getPreviousReleaseTag($output, $release),
+        ];
+
+        // Get release notes from GitHub API
+        $slug = $release->getLibrary()->getGithubSlug();
+        list($org, $repo) = explode('/', $slug);
+        $reponse = $client->getHttpClient()->post(
+            '/repos/'.rawurlencode($org).'/'.rawurlencode($repo).'/releases/generate-notes',
+            [],
+            json_encode($args)
+        );
+
+        $notes = ResponseMediator::getContent($reponse);
+        if (isset($notes['body'])) {
+            return $notes['body'];
+        }
+
+        throw new LogicException('Could not get release notes for ' . $release->getLibrary()->getName());
+    }
+
+    /**
+     * Get the name of the tag to use as the previous version for generating release notes
+     */
+    private function getPreviousReleaseTag(
+        OutputInterface $output,
+        LibraryRelease $release
+    ): string {
+        $version = $release->getVersion();
+        $major = $version->getMajor();
+        $minor = $version->getMinor();
+        $patch = $version->getPatch();
+
+        // Patch release (e.g. 1.2.3)
+        // Previous tag is x.y.z-1 (e.g. 1.2.2)
+        if ($patch != 0) {
```

Comment:
```suggestion
        if ((int) $patch !== 0) {
```
Should be doing this sort of thing?

Reply:
No need to cast but yes they should absolutely be `===` compared. Or better yet, check if they're `>` 0 which is more readable for this purpose. I've done the latter.